src/unittest/unittestScripts.ts

```diff
@@ -12,6 +12,22 @@ import base64
 import json
 import traceback
 
+from pathlib import Path
```

Comment:
Can you please wrap all this logic to a function and call it below before the main logic is triggered?

Reply:
wilco